### ISSUE TITLE
Delete book recursive

### ISSUE THREAD

**Issue by Sebastian1515** [moan0s/ILMO2#41]
When deleting a book all book instances, loan reminders etc. should be also deleted.

**Comment by Sebastian1515** [comment]
@moan0s is this useful?

**Comment by moan0s** [comment]
Yes, this could be useful. Currently the Book is protected from being deleted if there is a bookinstance, attatched
